lib/modules/dosomething/dosomething_kudos/dosomething_kudos.module

```diff
@@ -309,3 +309,22 @@ function dosomething_kudos_term_is_selected($reportback_item, $term) {
 function dosomething_kudos_disable_reactions($nid) {
   return !variable_get('dosomething_reportback_kudos_enabled', FALSE) || dosomething_helpers_get_variable('node', $nid, 'disable_reactions');
 }
+
+function dosomething_kudos_get_kudos_data_for_fid($fid, $nid) {
+  $disable_reactions = dosomething_kudos_disable_reactions($nid);
```

Comment:
We'd then have to still put duplicate code everywhere,

```
$disable = kudos_disable(nid)
if (!disable)
 get_kudos_data
```

with this its just 1 line of code if you want to grab Kudos data & forget about it